internal/engine/engine_state.go

```diff
@@ -15,13 +16,20 @@ type engineState struct {
 	manifestsToBuild  []model.ManifestName
 	currentlyBuilding model.ManifestName
 	completedBuilds   chan completedBuild
+
+	initialBuildCount   int
```

Comment:
Hmm, could there be a better name for this? I think a lot of people are going to be looking at `engineState`. Maybe a comment? The others are clear to me by name though. 👍